epic-games.js

```diff
@@ -3,16 +3,17 @@ import path from 'path';
 import { __dirname, stealth } from './util.js';
 const debug = process.env.PWDEBUG == '1'; // runs non-headless and opens https://playwright.dev/docs/inspector
 
-const URL_LOGIN = 'https://www.epicgames.com/login';
-const URL_CLAIM = 'https://www.epicgames.com/store/en-US/free-games';
+const URL_CLAIM = 'https://store.epicgames.com/store/en-US/free-games';
```

Comment:
Drop `/store` to match now.

Reply:
Did you check?
The log above showed it was waiting for
`https://store.epicgames.com/store/en-US/free-games`, but navigated to
`https://store.epicgames.com/en-US/free-games`.